analysis/taint/testdata/builtins_121/config.yaml

```diff
@@ -2,13 +2,8 @@
 taint-tracking-problems:
   -
     sources:
-      - package: "(main)|(command-line-arguments)|(selects-test[[:graph:]]*)$"
+      - package: "(builtins_121)|(main)|(command-line-arguments)|(git.amazon.com[[:graph:]]*)$"
```

Comment:
Looks like git is showing some wrong "file moves".

Reply:
That's weird, it's showing something similar for the selects-tests as well: https://github.com/awslabs/ar-go-tools/pull/66/files/15b61e6204d1b691aaf02d8c046d09f3ecf9c387#diff-629ad348bed03364019216de19f986a6be4f5dc2b67792e0278bc913fd863b21